v22.2/known-limitations.md

```diff
@@ -10,6 +10,89 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
 ## New limitations
 
+### DROP OWNED BY does not support drop functions
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
+
+In its current implementation, this statement does not drop functions. Users must drop their functions manually.
+
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/90476)
+
+{% comment %}
+    initial DROP OWNED BY implementation PR/issue:
+    https://github.com/cockroachdb/cockroach/pull/82936
+    https://github.com/cockroachdb/cockroach/issues/55381
+    doc work in progress (to link to after merge):
+    https://github.com/cockroachdb/docs/pull/15094/files
+    https://cockroachlabs.atlassian.net/browse/DOC-4770 
+    review: rafiss (or devadvocado)
+{% endcomment %}
+
+### DROP OWNED BY not supported where role has synthetic privileges
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
+
+In its current implentation, this operation cannot be performed for roles that have synthetic privileges (entries in `system.privileges`).
+
+Instead, you can use `REVOKE SYSTEM` for the relevant privileges the role has in `system.privileges`.
+
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/88149)
+
+{% comment %}
+    initial DROP OWNED BY implementation PR/issue:
+    https://github.com/cockroachdb/cockroach/pull/82936
+    https://github.com/cockroachdb/cockroach/issues/55381
+    doc work in progress (to link to after merge):
+    https://github.com/cockroachdb/docs/pull/15094/files
+    https://cockroachlabs.atlassian.net/browse/DOC-4770 
+    review: rafiss
+{% endcomment %}
+
+### Limitations for user-defined functions (UDFs)
+
+#### Limitations on use of UDFs
+
+User-defined functions are not currently supported in:
+
+- Expressions (column, index, constraint) in tables.
+- Views.
+- Other user-defined functions.
+- [CDC transformations](https://www.cockroachlabs.com/docs/v22.2/cdc-transformations).
+
+#### Limitations on expressions allowed within UDFs
+
+The following are not currently allowed within the body of a UDF:
+
+- Subqueries in statements.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/87291)
+
+- Mutation statements such as `INSERT`, `UPDATE`, `DELETE`, and `UPSERT`.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/87289)
+
+- Expressions with `*` such as `SELECT *`.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/90080)
+
+- CTEs (common table expressions).
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/92961)
+
+- Other user-defined functions.
```

Comment:
@mgartner Thanks for your write-up. Could you please review the updated section? We don't have issues for the bullets that were already in the docs. Would you be willing to try to find or create these? (Or let me know if there is any item for which you'd prefer not to have an issue.)

Reply:
Saw the other thread and will update to "_References to_ other user-defined functions."

Reply:
For "other user-defined functions" you can link to https://github.com/cockroachdb/cockroach/issues/88198 and https://github.com/cockroachdb/cockroach/issues/93049.

For "Expressions (column, index, constraint) in tables" and "views" you can link to https://github.com/cockroachdb/cockroach/issues/87699.

Reply:
Thanks!